packages/web-react/src/components/Dropdown/useDropdownAriaProps.ts

```diff
@@ -0,0 +1,36 @@
+import { Booleanish, ClickEvent } from '../../types';
+
+const NAME_ARIA_EXPANDED = 'aria-expanded';
```

Comment:
Maybe `ATTRIBUTE_ARIA_EXPANDED` would be more accurate.

Reply:
Fixed